### ISSUE TITLE
Multiple conversations for same bot

### ISSUE THREAD

**Issue by sschoeb** [MicrosoftDocs/msteams-docs#11689]
### Steps to reproduce

I don't know how that was possible, but I do have multiple conversations with my app, which should never be possible

### Expected behavior

Never have 2 conversations with the same bot.

### Actual behavior

![image](https://github.com/user-attachments/assets/ff5574ab-cbc2-4d08-872f-c35c20e5f4b4)

As you can see in the screenshot, I do have two conversations with the same app. These are separate conversations as they have different IDs:
```
a:167R9D5bp4Xqfy1CKxxbSMzShXsKhvSlGaQkQljiZrL8efAa-8cg1FgoVQXOmVDmsXzDfyi9NpgX8IYwHPI6attPdLcTKhylXIkTMd_q1TIiXesBtNjKoQm5dg9fOQ2_o
a:1ViotFCZ_wqnrKUUzJllQQjjNOBA1TlJI60jo55QCHw4KtNntd2OZc4pFsMXM3BlaxwLINToYaDiiuOaMGjsJ3JpLu4Vqfg_WvFxSCJrsdi_5tifM7xpWe6rMleBZ4qJj
```

As you can imagine, this ends up in a huge mess on our bot, as the assumption is, that we only have one possible conversation in teams.

### Error details

Teams version:
You have Microsoft Teams version 24257.204.3155.53.
You've got the latest version. It was last checked on 14.10.2024.
The client version is 49/24091221314.

Tested on Windows 11

**Comment by Meghana-MSFT** [comment]
@sschoeb - Could you please confirm if you started seeing this issue recently? Are you facing this issue with other bots as well? We checked this at our end by adding the bot in the below Team's version which is same as your version. We see only 1 chat being created with the bot.

You have Microsoft Teams version 24257.204.3155.53.
You've got the latest version. It was last checked on 14/10/2024.
The client version is 49/24091221314.

**Comment by sschoeb** [comment]
@Meghana-MSFT Yes, I see this since a few days (not exactly sure since when tbh). 

No, I just see it for this bot. Bot I'm no using others, therefore cannot answer this question properly.

**Comment by Meghana-MSFT** [comment]
@sschoeb - We raised a bug for this, we will keep you posted on the updates. Thank you.

**Comment by ChetanSharma-msft** [comment]
Hello @sschoeb,

Apologies for the delay in responding.
We’ve received updates from the Engineering team confirming that if there are bots with different IDs but the same name, it is expected behavior for both to appear in the chat list.

Please let us know if you have any further questions.

**Comment by sschoeb** [comment]
@ChetanSharma-msft and how is this possible? As this is the same app? How can I end up with two chats by installing the same app somehow again? I think this should never be possible right?
